verilog/tools/kythe/indexing_facts_tree_extractor_test.cc

```diff
@@ -3154,140 +3154,6 @@ TEST(PackageImportTest, PackageDirectMemberReference) {
   EXPECT_EQ(result_pair.right, nullptr) << *result_pair.right;
 }
 
-TEST(FactsTreeExtractor, ForLoopInitializations) {
-  constexpr int kTag = 1;  // value doesn't matter
```

Comment:
I removed this test because now we create the scope with random id and it can't be tested but this is covered in verification test with 2 or more tests.
We can keep a counter for the scopes we create for loops (keep our own counter instead of using random id).

Reply:
Suggested above, keep one counter per enclosing scope.